### PR TITLE
tests: tfm: Added cc3xx and oberon crypto backends to test variants

### PR DESCRIPTION
**Pull request by magnev**
- Added both cc3xx and oberon crypto backends to increase test coverage

Ref: NCSDK-17185

Signed-off-by: Magne Værnes <magne.varnes@nordicsemi.no>

test-sdk-nrf: sdk-nrf-PR-9340

### CODE REVIEW

File: tests/tfm/tfm_regression_test/testcase.yaml [hunk 1]
```diff
@@ -24,3 +24,11 @@ tests:
 
   tfm.regression_ipc_lvl2:
     timeout: 200
+
```
Comment (joerchan) [inline tests/tfm/tfm_regression_test/testcase.yaml:27]: One of these is a duplication of the tfm.regression_ipc_lvl2 configuration.
If we don't want to rely on the default value in this testcase file I suggest we remove `tfm.regression_ipc_lvl2`.

File: tests/tfm/tfm_regression_test/testcase.yaml [hunk 2]
```diff
@@ -24,3 +24,11 @@ tests:
```
Comment (joerchan) [inline tests/tfm/tfm_regression_test/testcase.yaml:24]: @magnev Should we consider both enabled, and both disabled also?

Comment (magnev) [reply tests/tfm/tfm_regression_test/testcase.yaml:24]: Yes, that would guarantee that we test all backend variants. What do you think @frkv ?

**Review by joerchan — APPROVED** [review]

**Review by frkv — APPROVED** [review]
LGTM

**Pull request merged**